src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAScoring.h

```diff
@@ -41,8 +41,11 @@ namespace OpenMS
          number is then weighted by the relative intensity of the transition
          (thus less is better here).
 
-    - massdiff score: computes the difference in ppm of the experimental signal
-         to the expected signal (thus less is better)
+    - massdiff score: computes the difference in ppm of the experimental signal. 
+         to the expected signal (thus less is better). 
+         sum(ppm_difference) / # transitions
+         Divide by the total number of transitions and is thus quite punnishing if a transition is missing
+         Also outputs a list of all the ppm differences, if signal is not found output -1.0
```

Comment:
try to build the OpenMS docs locally, and inspect what Doxygen makes of this. It will ignore manual linebreaks, so this will read differently. May need some adjustments.